packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -467,6 +491,20 @@ class CupertinoSlidingSegmentedControl<T extends Object> extends StatefulWidget
   /// Defaults to `EdgeInsets.symmetric(vertical: 2, horizontal: 3)`.
   final EdgeInsetsGeometry padding;
 
+  /// Determines whether segments in the segmented control show selected state.
+  ///
+  /// If true, segments in the control don’t show selected state and
+  /// don’t update the value of selectedSegmentIndex after tracking ends.
```

Comment:
Implementation details are not very helpful in public facing docs. What is this explaining to the user?